### ISSUE TITLE
文本遗漏

### ISSUE THREAD

**Issue by DandelionI** [dotnet/AspNetCore.Docs#21420]
[在此处输入反馈]
**Student 实体** 一项内，_有关详细信息，请参阅 F Core - 密钥。_ 中应为 _EF Core_

---
#### 文档详细信息

⚠ *请勿编辑此部分。 docs.microsoft.com ➟ GitHub 问题链接必须具有此部分。*

* ID: 7a54c97f-ddd4-b041-5e5f-a2df009ace50
* Version Independent ID: fa488196-1f98-08ea-adcc-17d9dac86727
* Content: [ASP.NET Core 中的 Razor Pages 和 Entity Framework Core - 第 1 个教程（共 8 个）](https://docs.microsoft.com/zh-cn/aspnet/core/data/ef-rp/intro?view=aspnetcore-5.0&tabs=visual-studio)
* Content Source: [aspnetcore/data/ef-rp/intro.md](https://github.com/dotnet/AspNetCore.Docs.zh-cn/blob/live/aspnetcore/data/ef-rp/intro.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-data**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
@guardrex  here's another no-loc. Once we solve it, notify the EF folks  at `https://github.com/dotnet/EntityFramework.Docs/issues/3068`
link above.

**Comment by Rick-Anderson** [comment]
@guardrex  closing since we no longer use meta-data. Do we need to add "EF Core"?

**Comment by guardrex** [comment]
Yes ... it can be added. We didn't at the time for some reason. It goes here ...

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/docfx.json#L69
